src/components/Home/ScrollIndicator.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+import React from 'react';
+import { ChevronDown } from 'lucide-react';
+
+const ScrollIndicator = () => {
+  const scrollToNextSection = () => {
+    // First try to find the why-join section
+    const whyJoinSection = document.getElementById('why-join');
+    if (whyJoinSection) {
+      whyJoinSection.scrollIntoView({
+        behavior: 'smooth',
+        block: 'start'
+      });
+      return;
+    }
+
+    // Fallback: scroll down by viewport height
+    const viewportHeight = window.innerHeight;
+    window.scrollTo({
+      top: viewportHeight,
+      behavior: 'smooth'
+    });
+  };
```

Comment:
Fallback here should be to navigate to the id. Something like `router.push('#why-join')` since we've applied smooth scrolling to the root of the document